INSTALL.md

```diff
@@ -20,6 +20,9 @@ To compile Coq yourself, you need:
 - [OCaml](https://ocaml.org/) (version >= 4.09.0)
   (This version of Coq has been tested up to OCaml 4.13.0)
 
+  The flambda version of the compiler is known to provide good
```

Comment:
Perhaps some more details here like the fact that you have to install `ocaml-variants` and `option-flambda` to get this working for more recent versions of OCaml?

Reply:
Sure, tho we risk that kind of work being out of date. Referering to the official opam documentation seems more robust to me.